packages/graphql-mesh-server/lib/fargate.ts

```diff
@@ -61,6 +61,14 @@ export interface MeshServiceProps {
    * SSM values to pass through to the container as secrets
    */
   secrets?: { [key: string]: ssm.IStringParameter | ssm.IStringListParameter };
+
+  /**
+   * ECS Secrets to pass through to the container as secrets
+   *
+   * The key values can be referenced from either SSM or Secrets manager
+   */
+  secretsV2?: { [key: string]: ecs.Secret };
```

Comment:
Can we name this something else? `secretsName` or similar?

Reply:
This isn't expecting a single secret but a key value pair of secrets. I'd like to keep it so it's flexible (can take in multiple secrets if necessary) and also slightly clear that `secrets` and `secretsV2` are closely related and the newer approach should be preferred.